microsoft-edge/progressive-web-apps-chromium/ux.md

```diff
@@ -0,0 +1,108 @@
+
+:::image type="complex" source="./media/pwa-in-apps-and-features-settings.png" alt-text="The list of installed apps on Windows also display PWAs" lightbox="./media/pwa-in-apps-and-features-settings.png":::
+   The list of installed apps on Windows also display PWAs  
+:::image-end:::  
+
+## Windows integration  
+
+PWAs are displayed like native apps on Windows. They appear in the Taskbar (where they can be pinned), in the Start Menu, or when switching between apps with `Alt`+`Tab`.  
+
+:::image type="complex" source="./media/windows-start-menu.png" alt-text="PWAs and native apps side by side in the Taskbar and Start Menu" lightbox="./media/windows-start-menu.png":::
+   PWAs and native apps side by side in the Taskbar and Start Menu  
+:::image-end:::  
+
+PWAs can also expose common tasks to users as shortcuts in the Windows Taskbar contextual menu. To learn about how to define shortcuts in your PWA, navigate to [Define app shortcuts][AppShortcutsFeature].  
+
+:::image type="complex" source="./media/pwa-shortcuts-in-taskbar.png" alt-text="Common tasks are listed in the Taskbar contextual menu" lightbox="./media/pwa-shortcuts-in-taskbar.png":::
+   Common tasks are listed in the Taskbar contextual menu  
+:::image-end:::  
+
+PWAs can also display notifications in the operating system's own notification service. This helps users re-engage with your app. Navigate to [Re-engage users with notifications, push messages, and badges][PushNotficationsFeature] to learn more.  
+
+:::image type="complex" source="./media/windows-action-center.png" alt-text="Notifications in Windows Action Center" lightbox="./media/windows-action-center.png":::
+   Notifications in Windows Action Center  
+:::image-end:::  
+
+### Starting PWAs when the user logs in  
+
+PWAs can launch automatically when the users into Windows. This capability allows a user to engage with the app as soon as the user logs into the OS.  
+
+To turn on the `Start app when you sign in` feature for an installed PWA:  
+
+1.  Open Microsoft Edge.
+1.  Navigate to `edge://apps`.
+1.  Hover on your app.
+1.  Open the contextual menu (right-click) and then select **Start app when you sign in**.
+
+:::image type="complex" source="./media/turn-on-run-on-os-login-flag.png" alt-text="Use the contextual menu to turn on the Start app when you sign in feature in Microsoft Edge" lightbox="./media/turn-on-run-on-os-login-flag.png":::
+   Use the contextual menu to turn on the **Start app when you sign in** feature in Microsoft Edge
+:::image-end:::  
+
+Upon installation of an app, users are also given a chance to turn on the feature in the post-installation dialog.  
+
+:::image type="complex" source="./media/post-install-run-on-os-login.png" alt-text="The post-installation dialog automatically opens once an app is installed" lightbox="./media/post-install-run-on-os-login.png":::
+   The post-installation dialog automatically opens once an app is installed  
+:::image-end:::  
+
+## App info menu  
```

Comment:
I believe there is some work underway to change the appearance (and maybe the behavior) of the … menu. I’m not sure who owns that work though. It might be Jason.